integ-tests/src/main/java/com/google/testdata/TestDataHelper.java

```diff
@@ -75,22 +77,22 @@ public static void generateDbTablePairs(Connection connection, int dbTablePairsC
     final String dbData = getSql(TESTDATA_SQL_PATH + "columns_data_1.sql");
     final String tableData = getSql(TESTDATA_SQL_PATH + "columns_data_2.sql");
 
-    List<String> dbTableQueries = new ArrayList<>();
+    List<String> sqlQueries = new ArrayList<>();
     while (dbTablePairsCount > 0) {
-      String dbName = format("test_%s_%s", nanoTime(), dbTablePairsCount);
-      String tableName = format("test_%s_%s", nanoTime(), dbTablePairsCount);
+      String dbName = getRandomDbName();
+      String tableName = getRandomTableName();
 
-      dbTableQueries.add(format(dbData, dbName));
-      dbTableQueries.add(format(tableData, dbName, tableName));
+      sqlQueries.add(format(dbData, dbName));
+      sqlQueries.add(format(tableData, dbName, tableName));
       dbTablePairsCount--;
     }
 
-    executeQueries(connection, dbTableQueries);
+    executeQueries(connection, sqlQueries);
 
     LOGGER.info(
         format(
             "Generated %s new DB and Table pair(s):\n%s",
-            dbTableQueries.size() / 2, Joiner.on("\n").join(dbTableQueries)));
+            sqlQueries.size() / 2, Joiner.on("\n").join(sqlQueries)));
```

Comment:
You can use [`System.lineSeparator()`](https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#lineSeparator--) to get the OS specific line separator.

Reply:
Done.
Changed all `\n` to `lineSeparator()`